library/src/nvcc_detail/hipsparse.cpp

```diff
@@ -3628,6 +3628,246 @@ hipsparseStatus_t hipsparseZnnz(hipsparseHandle_t         handle,
                                                      nnzTotalDevHostPtr));
 }
 
+hipsparseStatus_t hipsparseSdense2csr(hipsparseHandle_t         handle,
+                                      int                       m,
+                                      int                       n,
+                                      const hipsparseMatDescr_t descr,
+                                      const float*              A,
+                                      int                       ld,
+                                      const int*                nnzPerRow,
+                                      float*                    csrVal,
+                                      int*                      csrRowPtr,
+                                      int*                      csrColInd)
+{
+    return hipCUSPARSEStatusToHIPStatus(cusparseSdense2csr(
+        (cusparseHandle_t)handle, m, n, descr, A, lda, nnzPerRow, csrVal, csrRowPtr, csrColInd));
```

Comment:
Do you need a cast here for the matrix descriptor (and for the rest)? i.e. (const cusparseMatDescr_t)descrA?